controllers/internalrequest/adapter_test.go

```diff
@@ -84,6 +112,53 @@ var _ = Describe("PipelineRun", Ordered, func() {
 		})
 	})
 
+	Context("When calling EnsureRequestIsAllowed", func() {
+		AfterEach(func() {
+			deleteResources()
+		})
+
+		BeforeEach(func() {
+			createResources()
+		})
+
+		It("should allow any request with spec.allowList being empty", func() {
```

Comment:
should allow any request when spec.allowList is empty?

Reply:
Yeah, that's a good question. Originally I blocked any namespace not in the allow list, so an empty list would block everything. I thought that maybe we wanted to have a way to allow everything at some point, so I changed the behavior to allow everything for an empty list but the moment the list have a value, it will only allow requests for matching namespaces.

What should we do, @scoheb?

Reply:
The safest thing to do, from a security POV, would be `an empty list would block everything`